src/layout/mutation.ts

```diff
@@ -37,9 +36,15 @@ export function start(): void {
     };
 }
 
+export function observe(node: Node): void {
+  let observer = window["MutationObserver"] ? new MutationObserver(measure(handle) as MutationCallback) : null;
```

Comment:
Why do you create a new observer each time here rather than once (as before the change)? Would be great to add a comment with your explanation.

Reply:
I'll add the comment. Basically each dom tree within shadow DOM needs a different observer since uber observer no longer works within shadow DOM.

Reply:
Added a comment